app/controllers/api/android_apps_controller.rb

```diff
@@ -12,14 +12,14 @@ def index
   end
 
   def create
-    render plain: "Send me an array, dumbass", status: 400 and return unless params[:apps].is_a?(Array)
+    render plain: "Sir, please give array", status: 400 and return unless params[:apps].is_a?(Array)
 
     updated_apps = @user.update_apps(params['apps'])
     render :json => updated_apps.to_json
   end
 
   def batch_delete
-    render plain: "Send me a uid param, dumbass!", status: 400 and return unless params[:uid]
+    render plain: "Sir, pleasee give uid param. Please sir.", status: 400 and return unless params[:uid]
 
```

Comment:
@rohitpaulk us back then writing apis.

Reply:
haha

Reply:
I'd make this `Saar` to adhere to our coding guidelines